packages/ui/lib/components/TrackTable/Cells/TitleCell.tsx

```diff
@@ -71,7 +71,7 @@ const TitleCell: React.FC<CellProps<Track> & TrackTableExtraProps<Track>> = ({
         withAddToQueue={Boolean(onAddToQueue)}
         withAddToFavorites={Boolean(onAddToFavorites)}
         withAddToPlaylist={Boolean(onAddToFavorites)}
-        withAddToDownloads={Boolean(onAddToDownloads)}
```

Comment:
This should stay this way, if there's not `onAddToDownloads` function then it will disable the button automatically.